src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/SmeltersPickaxe.java

```diff
@@ -27,7 +27,7 @@ public SmeltersPickaxe(Category category, SlimefunItemStack item, RecipeType rec
     @Override
     public ToolUseHandler getItemHandler() {
         return (e, tool, fortune, drops) -> {
-            if (MaterialCollections.getAllOres().contains(e.getBlock().getType()) && !BlockStorage.hasBlockInfo(e.getBlock())) {
+            if (SlimefunTag.SMELTERS_PICKAXE_BLOCKS.isTagged(e.getBlock().getType()) && !BlockStorage.hasBlockInfo(e.getBlock())) {
```

Comment:
This could use some refactoring too.

Reply:
Couldn't really think how this would be improved.

Reply:
> Couldn't really think how this would be improved.

I mean like separating into multiple lines instead of 1 very long line.

Reply:
No.